src/commcare_cloud/environment/main.py

```diff
@@ -105,9 +105,16 @@ def proxy_config(self):
 
     @memoized_property
     def users_config(self):
-        with open(self.paths.get_users_yml(self.meta_config.users)) as f:
-            users_json = yaml.load(f)
-        return UsersConfig.wrap(users_json)
+        user_groups_from_yml = self.meta_config.users
+        absent_users = []
+        present_users = []
+        for user_group_from_yml in user_groups_from_yml:
+            with open(self.paths.get_users_yml(user_group_from_yml)) as f:
+                user_group_json = yaml.load(f)
+                present_users += user_group_json['dev_users']['present']
+                absent_users += user_group_json['dev_users']['absent']
+        all_users_json = {'dev_users': {'absent': absent_users, 'present': present_users}}
```

Comment:
Something I thought of here is can we validate that there's no overlap between `absent_users` and `present_users`? With the extra level of expressiveness that we get from this change, we could also inadvertently mess that up, and until we have a well defined behavior for what that would mean, I think it's good to just fail and say that we need to fix our users files before continuing.

(Make sure that wherever this validation happens, a failure in it results in a `nosetests` failure as well as a `cchq <env> validate-environment-settings` failure.)